balena/models/organization.py

```diff
@@ -161,3 +164,241 @@ def remove(self, org_id):
+                endpoint=self.settings.get('pine_endpoint'), login=True
+            )['d']
+
+            if not org:
+                raise exceptions.OrganizationNotFound(handle_or_id)
+
+            params['eq'] = org['id']
+
+        return self.base_request.request(
+            'organization_membership', 'GET', params=params,
+            endpoint=self.settings.get('pine_endpoint'), login=True
+        )['d']
+
+    def get(self, membership_id):
+        """
+        Get a single organization membership.
+
+        Args:
+            membership_id (str): organization id.
+
+        Returns:
+            Organization membership.
+
+        Raises:
+            OrganizationNotFound: if organization couldn't be found.
+
+        Examples:
+            >>> balena.models.organization.membership.get(3014)
+            '[{'id': 17608, 'created_at': '2017-08-03T11:16:03.022Z', 'user': {'__id': 22294, '__deferred': {'uri': '/resin/user(@id)?@id=22294'}}, 'is_member_of__organization': {'__id': 3014, '__deferred': {'uri': '/resin/organization(@id)?@id=3014'}}, 'organization_membership_role': {'__id': 3, '__deferred': {'uri': '/resin/organization_membership_role(@id)?@id=3'}}, '__metadata': {'uri': '/resin/organization_membership(@id)?@id=17608'}}]'
+
+        """
+
+        params = {
+            'filter': 'id',
+            'eq': membership_id
+        }
+
+        org_membership = self.base_request.request(
+            'organization_membership', 'GET', params=params,
+            endpoint=self.settings.get('pine_endpoint'), login=True
+        )['d']
+
+        if len(org_membership) > 0:
+            return org_membership[0]
+        else:
+            raise exceptions.OrganizationMembershipNotFound(membership_id)
+
+
+class OrganizationMembershipTag(BaseTag):
+    """
+    This class implements organization membership tag model for balena python SDK.
+
+    """
+
+    def __init__(self):
+        super(OrganizationMembershipTag, self).__init__('organization_membership')
+
+    def get_all_by_organization(self, handle_or_id):
+        """
+        Get all organization membership tags for an organization.
+
+        Args:
+            handle_or_id (str): organization handle (string) or id (number).
+
+        Returns:
+            list: organization membership tags.
+
+        Raises:
+            OrganizationNotFound: if organization couldn't be found.
+
+        Examples:
+            >>> balena.models.organization.membership.tag.get_all_by_organization(3014)
+            [{'id': 991, 'organization_membership': {'__id': 17608, '__deferred': {'uri': '/resin/organization_membership(@id)?@id=17608'}}, 'tag_key': 'mTag1', 'value': 'Python SDK 1', '__metadata': {'uri': '/resin/organization_membership_tag(@id)?@id=991'}}]
+
+        """
+
+        org_id = handle_or_id
+
+        if not is_id(handle_or_id):
+            params1 = {
+                'filter': 'handle',
+                'eq': handle_or_id
```

Comment:
Optional: Since we only use the org id, can we also add a `$select` that selects just that?